tests/storage/cdi_upload/conftest.py

```
@@ -99,6 +70,34 @@ def uploaded_dv_scope_class(unprivileged_client, namespace, storage_class_name_s
             yield dv
 
 
+@pytest.fixture()
```

Ruff (0.16.0)

[warning] Use `@pytest.fixture` over `@pytest.fixture()`

Remove parentheses

(PT001)